db/migrate/20230415003305_add_keys_to_courses.rb

```diff
@@ -0,0 +1,5 @@
+class AddKeysToCourses < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
Combine all the keys whether it's a reference or an index of one module in a single file.

```ruby
add_reference :courses, :instructor, foreign_key: { to_table: :users }, null: false
add_reference :courses, :institution, foreign_key: true, null: false
add_index :courses, :instructor, name: "fk_course_users"
```

Reply:
Should I delete the other related migration files and keep all this in a single file?